internal/api/genapi/parse.go

```diff
@@ -40,24 +40,41 @@ func parsePBs() {
 		// Evaluate above, populate below.
 		in.generatedStructure.pkg = packageFromFullName(desc.FullName())
 		in.generatedStructure.name = string(desc.Name())
+		var attributesFilter []string
 		for i := 0; i < desc.Fields().Len(); i++ {
 			fd := desc.Fields().Get(i)
+			opts := fd.Options().(*descriptorpb.FieldOptions)
+			isAttribute := false
+			if proto.GetExtension(opts, protooptions.E_Subtype).(string) != "" {
+				isAttribute = true
+			}
+			if isAttribute {
+				// Only parse attributes oneOf once and not for each option
+				if strutil.StrListContains(attributesFilter, "Attributes") {
+					continue
+				}
+			}
```

Comment:
I wonder if we could simplify this now, by simply checking if the subtype is different from `"default"`, and if so, excluding the field. What do you think?